src/index.js

```diff
@@ -9,8 +9,13 @@ import { createWebhook, retrieveWebhook, listWebhooks, toggleWebhook } from './w
 
 export default class Paymongo {
   constructor (secret) {
-    if (Paymongo.instance instanceof Paymongo) return Paymongo.instance;
     if (!secret) throw new Error('API key is required!');
+
+    if (Paymongo.instance instanceof Paymongo) {
```

Comment:
This can be done in a single `if` for cleaner code. 

```js
if (Paymongo.instance instanceof Paymongo && Paymongo.instance.secret === secret)
``` 

Kindly update, thanks.